news/12125-added-conda._deprecated

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Added new `conda._deprecated` decorator and `conda._deprecated.module` functions for easier/standardized deprecation. (#12125)
```

Comment:
Do we need to publish this as `conda._deprecated`? I assumed we'd announce the existence of `conda.deprecations` instead, so we don't have to worry what happens in `conda.__init__` long-term?
```suggestion
* Added a new `conda.deprecations` module for easier/standardized deprecation, including a `deprecated` decorator and functions to mark modules for deprecation.  (#12125)
```

Reply:
oops, this was just an oversight on my part